### ISSUE TITLE
Improve python extractor: specific problem

### ISSUE THREAD

**Issue by KarlaRosas** [Jlebours/VeriteTerrain_Editor#4]
We have a problem with the URL : https://en.wikipedia.org/wiki/Comparison_of_digital_SLRs
On the website there is a data table and our extractor finds 8 data tables.

![image](https://user-images.githubusercontent.com/72513172/99446062-c6713e00-291e-11eb-8312-4878f878f744.png)

Extracteur python:

![image](https://user-images.githubusercontent.com/72513172/99446196-fc162700-291e-11eb-90cd-70c4b9f373c5.png)

**Comment by KarlaRosas** [comment]
In a first analysis, I can conclude that the first table generated in the python extractor is the table corresponding to the website of the URL.
![image](https://user-images.githubusercontent.com/72513172/99450321-ebff4700-2920-11eb-9c50-aeccb02e0b73.png)

From the second table generated by the python extractor, there are inconsistencies in the data, since I cannot visualize it on the website

**Comment by KarlaRosas** [comment]
To find the problem, I decided to directly analyze the HTML code https://en.wikipedia.org/wiki/Comparison_of_digital_SLRs
![image](https://user-images.githubusercontent.com/72513172/99640844-bd30c000-2a49-11eb-8193-23d987cfbcb8.png)


I found the correct table class on the website "wikitable sortable" 


The problem at the code level is that it does not consider this class

![image](https://user-images.githubusercontent.com/72513172/99641094-1862b280-2a4a-11eb-8c97-e2278e82b602.png)

**Comment by KarlaRosas** [comment]
I'm going to get the summary of the number of tables created with the current version of the extract.
[Summary1.txt](https://github.com/Jlebours/VeriteTerrain_Editor/files/5565591/Summary1.txt)

I am going to add the option to also consider "wikitable sortable" in the code and I am going to compare the results with the new summary

![image](https://user-images.githubusercontent.com/72513172/99644428-36caad00-2a4e-11eb-8097-d0ee7fd3822f.png)

Summary 2
[Summary2.txt](https://github.com/Jlebours/VeriteTerrain_Editor/files/5565601/Summary2.txt)

**Comment by KarlaRosas** [comment]
![image](https://user-images.githubusercontent.com/72513172/99650518-ce7fc980-2a55-11eb-9190-f39dae3ab13e.png)


When I got summary 1 and summary 2, I compared them and found table differences in other urls. The change modified the results in other Urls.
I found the problem but now I have to find a way to solve it. I have a question, what are the correct results?

**Comment by KarlaRosas** [comment]
When I did the modification in the code to find only the active table in the referenced URL, I noticed that the other URLs changed
Seeing the changes started to be difficult, so I decided to create a Dashboard to quickly analyze what happens with the data

![image](https://user-images.githubusercontent.com/72513172/99901094-5bc05980-2cb4-11eb-86c6-7bda9dfea84f.png)

So far I still can't wet the extractor

**Comment by KarlaRosas** [comment]
I re-analyzed the problem.
The python extractor was exporting 8 tables but on the website we could only see 1.

![image](https://user-images.githubusercontent.com/72513172/99909855-26832e00-2ceb-11eb-8611-b77e8b09e9ee.png)

I found the problem, on the site there are 7 more tables but they are hidden.
They contain a special class: 

![image](https://user-images.githubusercontent.com/72513172/99909845-1408f480-2ceb-11eb-8213-70093f21125f.png)

**Comment by KarlaRosas** [comment]
Our python extractor finds all the wikitable tables, (there are 8) but 7 are contained in the hidden class tables
<table class = "navbox mw-collapsible autocollapse">

I found a solution, exclude the hidden tables and then extract the remaining tables, that is, the visible tables.
![image](https://user-images.githubusercontent.com/72513172/99909945-a3160c80-2ceb-11eb-9da8-3be35b29509a.png)

**Comment by KarlaRosas** [comment]
I did the test with the code proposed above and I achieved the goal.
Now the extractor shows the visible tables of the wikitable class, wikitable sortable and sortable wikitable from a wikipedia website.

Analyze the results in a data analytics dashboard that I created, and effectively. version 1 and version 3, only change in the URL Comparison_of_digital_SLRs. Now we only get a table (visible table)

![image](https://user-images.githubusercontent.com/72513172/99910050-40714080-2cec-11eb-8696-2914fb47d593.png)

Data Analytics:  https://datastudio.google.com/reporting/6243c97a-7e9a-4a79-a67f-4f93d3a1d1c3
